index.Rmd

```diff
@@ -277,6 +277,11 @@ if(knitr::is_html_output())
 ```{r, echo=FALSE, include=!knitr::is_latex_output(), purl=FALSE}
 dev_version <- TRUE
 ```
+<!-- include=FALSE for PDF sending to CRC -->
+```{block, purl=FALSE, include=knitr::is_html_output()}
+This is the [website](https://moderndive.com/) for *Statistical Inference via Data Science:  A ModernDive into R and the Tidyverse*! Visit the [GitHub repository for this site](https://github.com/moderndive/ModernDive_book), find the book on [Amazon](https://www.amazon.com/Statistical-Inference-via-Data-Science/dp/0367409828/), or purchase it at [CRC Press](https://www.crcpress.com/Statistical-Inference-via-Data-Science-A-ModernDive-into-R-and-the-Tidyverse/Ismay-Kim/p/book/9780367409821) using promo code: **ASA18**.
```

Comment:
```suggestion
This is the [website](https://moderndive.com/) for *Statistical Inference via Data Science:  A ModernDive into R and the Tidyverse*! Visit the [GitHub repository for this site](https://github.com/moderndive/ModernDive_book), find the book on [Amazon](https://www.amazon.com/Statistical-Inference-via-Data-Science/dp/0367409828/), or purchase it at [CRC Press](https://www.crcpress.com/Statistical-Inference-via-Data-Science-A-ModernDive-into-R-and-the-Tidyverse/Ismay-Kim/p/book/9780367409821). Feel free to use promo code **ASA18** for a discount on the price.
```

Reply:
@rudeboybert The sentence was getting a little long so I broke it into two sentences instead to make it a little clearer. Feel free to commit and merge if that looks OK to you.